pulley/src/lib.rs

```diff
@@ -811,10 +811,16 @@ macro_rules! for_each_extended_op {
             fadd32 = Fadd32 { operands: BinaryOperands<FReg> };
             /// `low32(dst) = low32(src1) - low32(src2)`
             fsub32 = Fsub32 { operands: BinaryOperands<FReg> };
+            /// `low128(dst) = low128(src1) - low128(src2)`
+            vsub32x4 = Vsub32x4 { operands: BinaryOperands<VReg> };
```

Comment:
Mind using `f32x4` in the opcodes (e.g. `vsubf32x4`) to clearly indicate that it's for floats and not integers?